site/docs/community.md

```diff
@@ -32,106 +32,71 @@ Thanks for your interest in contributing to cssnano! Please see our
-    <td align="center"><a href="https://iankwalter.com"><img src="https://avatars0.githubusercontent.com/u/122028?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Ian Walter</b></sub></a><br /><a href="https://github.com/cssnano/cssnano/commits?author=ianwalter" title="Code">💻</a></td>
-    <td align="center"><a href="http://dave.mn"><img src="https://avatars3.githubusercontent.com/u/342029?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Dave McNally</b></sub></a><br /><a href="#design-davemcnally" title="Design">🎨</a></td>
-    <td align="center"><a href="https://badcss.smithua.com/"><img src="https://avatars1.githubusercontent.com/u/1473254?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Aleksandr Fesenko</b></sub></a><br /><a href="#platform-smithua" title="Packaging/porting to new platform">📦</a></td>
-    <td align="center"><a href="https://github.com/wongjn"><img src="https://avatars2.githubusercontent.com/u/11310624?v=4?s=100" width="100px;" alt=""/><br /><sub><b>wongjn</b></sub></a><br /><a href="https://github.com/cssnano/cssnano/commits?author=wongjn" title="Code">💻</a></td>
-    <td align="center"><a href="https://genemecija.com"><img src="https://avatars2.githubusercontent.com/u/39816902?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Gene Mecija</b></sub></a><br /><a href="https://github.com/cssnano/cssnano/issues?q=author%3Agenemecija" title="Bug reports">🐛</a> <a href="https://github.com/cssnano/cssnano/commits?author=genemecija" title="Code">💻</a> <a href="https://github.com/cssnano/cssnano/commits?author=genemecija" title="Tests">⚠️</a></td>
-    <td align="center"><a href="https://www.ludofischer.com"><img src="https://avatars1.githubusercontent.com/u/43557?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Ludovico Fischer</b></sub></a><br /><a href="#maintenance-ludofischer" title="Maintenance">🚧</a></td>
-  </tr>
-  <tr>
-    <td align="center"><a href="https://github.com/MapTo0"><img src="https://avatars.githubusercontent.com/u/5821279?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Martin</b></sub></a><br /><a href="https://github.com/cssnano/cssnano/commits?author=MapTo0" title="Code">💻</a></td>
-    <td align="center"><a href="https://github.com/IronGeek"><img src="https://avatars.githubusercontent.com/u/1383932?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Jakka Prihatna</b></sub></a><br /><a href="https://github.com/cssnano/cssnano/issues?q=author%3AIronGeek" title="Bug reports">🐛</a> <a href="https://github.com/cssnano/cssnano/commits?author=IronGeek" title="Code">💻</a> <a href="https://github.com/cssnano/cssnano/commits?author=IronGeek" title="Tests">⚠️</a></td>
-  </tr>
-</table>
-
-<!-- markdownlint-restore -->
-
-<!-- prettier-ignore-end -->
-
-<!-- ALL-CONTRIBUTORS-LIST:END -->
-
-This project follows the [all-contributors](https://github.com/kentcdodds/all-contributors) specification. Contributions of any kind welcome!
+Thanks goes to these wonderful people:
```

Comment:
I've checked there are 66 people listed, 65 on the website already + jamesgeorge007 added in this PR